Formula/sslyze.rb

```diff
@@ -1,9 +1,24 @@
 class Sslyze < Formula
+  include Language::Python::Virtualenv
+
   desc "SSL scanner"
   homepage "https://github.com/nabla-c0d3/sslyze"
-  url "https://github.com/nabla-c0d3/sslyze/archive/release-0.12.tar.gz"
-  version "0.12.0"
-  sha256 "5b3220d42cb66067b18d9055a2234252d849090e9fba660af52a3da18fa8a899"
+
+  stable do
+    url "https://files.pythonhosted.org/packages/6a/0a/7cbc2538a2e8d08dee48f290445c48cc817c4bdd978709b95526aa8025b0/SSLyze-0.13.6.tar.gz"
+    sha256 "5aa0e159cd0c118b6ae99ec12095707e5a7bd53c13dceeadd153ce767d941b66"
+
+    # pypi version has prebuilt binaries
+    resource "nassl" do
+      url "https://github.com/nabla-c0d3/nassl/archive/0.13.7.tar.gz"
+      sha256 "98512ac55d38d0839f795248f321ad9bf6971a3c0fb7224e84ad4f5c81a5bbcb"
+    end
+
+    resource "openssl" do
+      url "https://github.com/PeterMosmans/openssl.git",
```

Comment:
probably the last to have enough old ciphers to not break the build or something ...